src/services/scan_form_service.js

```diff
@@ -8,17 +8,49 @@ export default (easypostClient) =>
 
     /**
      * Create a ScanForm.
-     * @param {object} prams
+     * @param {object} params
      * @returns {ScanForm}
      */
     static async create(params) {
-      try {
-        // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
-        const response = await easypostClient.post(this._url, params);
+      const url = this._url;
 
-        return this.convertToEasyPostObject(response.body);
-      } catch (e) {
-        return Promise.reject(e);
+      // wraps up params in `shipments` if the user didn't do it
+      // turn a list of shipment objects into a map of shipment ids
+      if (params.shipments) {
+        // eslint-disable-next-line no-param-reassign
+        params.shipments = params.shipments.map((s) => {
+          if (typeof s === 'string') {
+            return { id: s };
+          }
+          return { id: s.id };
```

Comment:
```suggestion
        params.shipments = params.shipments.map((shipment) => {
          if (typeof shipment === 'string') {
            return { id: shipment };
          }
          return { id: shipment.id };
```
nit: `s` is a bad naming for future travelers